docs/kuraKapuaDocs.md

```diff
@@ -144,10 +144,10 @@ After that go to **Permissions** tab and click **Add**, select **ALL** in every
 
 Now we have everything we need to connect Kura to Kapua.  
 
-## Downloading and cofiguring Kura
+## Downloading and configuring Kura
 
-Second part of this guide describes how to configure Raspberry Pi, but you can also use BBB (BeagleBone Black) or Intel Edison [as described on Kura download page](http://www.eclipse.org/kura/downloads.php?).
-Because there already exists guide for [installing Kura on Raspberry Pi](https://eclipse.github.io/kura/intro/raspberry-pi-quick-start.html) we will not repeat the procedure here. When you finish, come back and follow procedure bellow. In this example wi will assume that Raspberry Pi has IP: 192.168.1.11 and our PC 192.168.1.10.
+The second part of this guide describes how to configure Raspberry Pi, but you can also use BBB (BeagleBone Black) or Intel Edison [as described on Kura download page](http://www.eclipse.org/kura/downloads.php?).
```

Comment:
"The second part" is not gramatically correct. Please change it back to "Second part...."